my_game.py

```diff
@@ -143,11 +143,13 @@ def on_update(self, delta_time):
             if self.invincibility_timer < 3:
                 # Visible
                 if self.alpha == 0:
-                    self.alpha = 255
+                    self.alpha = 155
                     self.center_x = PLAYER_START_X
                     self.center_y = PLAYER_START_Y
                     self.change_y = 0
                     self.change_x = 0
+            if self.invincibility_timer < 0.1:
+                self.alpha = 255
```

Comment:
Looks to me like two things will happen if the _invincibility_timer_ is 0. You should probably use an _if_ _else_. Don't you agree? Otherwise, you must add some comments to explain what you are doing in these lines.

Reply:
Yes, your right. I'll change it.